fluentd/map.jinja

```diff
@@ -30,4 +37,4 @@
         'destination_path': '/etc/init/fluentd.conf',
         'source_path': 'fluentd.upstart',
     }
-} %}
+}, grain='init', merge=salt.pillar.get('fluentd:init_service')) %}
```

Comment:
Just to double-clarify my understanding - this wasn't working before, right? Since at least the `grain` arg was needed.